### PR TITLE
chore: rename branch master to main

### PR DESCRIPTION
**Pull request by loicsay**
**⚠️ Merge this PR after changing `master` branch name to `main`**
[Guidelines here](https://github.com/github/renaming)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ifdef VERSION
 VERSION := $(VERSION)
 else
-VERSION := master
+VERSION := main
```
Comment (loicsay) [inline Makefile:4]: Not sure about that @clemfromspace 
What's the impact of changing the version name?

Comment (loicsay) [reply Makefile:4]: I didn't find any in the codebase 🤔 (but found an oversight in the Makefile)

File: Makefile [hunk 2]
```diff
@@ -16,7 +16,7 @@ docs:
 .PHONY: docs-commands-data
 docs-commands-data: docs
 	git -C docs pull
-	git -C docs checkout master
+	git -C docs checkout main
```
Comment (clemfromspace) [inline Makefile:19]: This is for the documentation repo, AFAIK they still are on the `master` branch.

**Review by clemfromspace — CHANGES_REQUESTED** [review]
One comment :)

**Pull request merged**